readthedocs/proxito/tests/handler_404_urls.py

```diff
@@ -21,18 +24,15 @@
 def map_proxito_path(view_func):
     @wraps(view_func)
     def inner_view(request, *args, **kwargs):
-        return view_func(
-            request,
-            proxito_path=request.path,
-        )
+        try:
+            return view_func(
+                request,
+                proxito_path=request.path,
+            )
+        except Http404 as e:
+            return proxito_handler404(request, exception=e)
 
     return inner_view
 
 
 handler404 = map_proxito_path(ServeError404.as_view())
-
-# Patch URLs that call the `fast_404` view directly.
-for pattern in urlpatterns:
-    if getattr(pattern, "name", None) == "docs_detail_directory_indexing":
-        pattern.callback = handler404
-        break
```

Comment:
We don't have a URL named `docs_detail_directory_indexing`, so not sure what this was about.